_includes/head.liquid

```diff
@@ -2,6 +2,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="robots" content="noindex">
```

Comment:
This will prevent google from indexing all environments i.e. staging AND production... You should add a liquid tag to conditionally insert this only when not production. There's an example here: https://github.com/AusDTO/dta-website/blob/develop/_includes/head.html#L100. However the dta-website has the site.url available at build time, currently content-guide doesnt, so you'll also have to specify that as well so that bin/cibuild.sh has it at build time....